pylint/testutils/unittest_linter.py

```diff
@@ -15,7 +15,7 @@
 class UnittestLinter:
     """A fake linter class to capture checker messages."""
 
-    # pylint: disable=unused-argument
+    # pylint: disable=unused-argument,duplicate-code
```

Comment:
Here the whole ``add_message`` function is duplicated from PyLinter so the disable is required but we have a useless suppression warning